### PR TITLE
fix: eslint peer deps list

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

fix all required packages are defined in peer deps.

#### 🤔 Why

Unable to perform full linting
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: docs/publishing.md [hunk 1]
```diff
@@ -10,23 +10,27 @@ sidebar_label: Publishing Packages
 
 1. Make changes and commit them to origin/branch
 2. Ensure your `git status` is porcelian
-3. On your local machine `npm run version`
+3. On your local machine
+   `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>] | from-git]`
```
Comment (estherthetesteramido) [inline docs/publishing.md:14]: This isn't correct... why are you bypassing the lerna versioning script?

File: docs/publishing.md [hunk 2]
```diff
@@ -10,23 +10,27 @@ sidebar_label: Publishing Packages
...
 4. Commit the automatically generated changes
 5. Raise a PR to origin/master
 6. The CI pipeline will then publish the changes for you
 
 ## What registry are we using?
 
-All of our packages are opensource and can be found under the @amidostacks organisation on [npm](https://www.npmjs.com/settings/amidostacks/packages).
+All of our packages are opensource and can be found under the @amidostacks
+organisation on [npm](https://www.npmjs.com/settings/amidostacks/packages).
```
Comment (estherthetesteramido) [inline docs/publishing.md:22]: Wrong URL - this is authenticated. Try: https://www.npmjs.com/~amidostacker

File: docs/publishing.md [hunk 3]
```diff
@@ -38,38 +42,59 @@ packages/
...
+   `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>] | from-git]`
+   ```js
+   // for example
+   npm version patch -m "Upgrade to %s for reasons"
+   ```
+2. `package.json` & `package-lock.json` version is changed
+3. the changes are to be merged to master (CHANGELOG.md, version and gitHead,
+   and the commit is tagged as a release in Github)
+4. in the pipeline, this triggers lerna to look for difference in package
+   version and the registry
```
Comment (estherthetesteramido) [inline docs/publishing.md:79]: Please see https://github.com/amido/stacks-webapp-template/pull/109

File: docs/publishing.md (hunk 1 above)
Comment (naren-bellala) [reply docs/publishing.md]: Reverted

File: docs/publishing.md (hunk 2 above)
Comment (naren-bellala) [reply docs/publishing.md:27]: I did not change anything here

**Comment by naren-bellala** [comment]
> Merge #109 first then have a go through please :)

Done

**Pull request merged**